internal/provider/resource_kubernetes_cloud_test.go

```diff
@@ -14,6 +14,12 @@ import (
 )
 
 func TestAcc_ResourceKubernetesCloud(t *testing.T) {
+	// Note (alesstimec): Skipping this test, because the default
+	// hosted cloud tf provider adds is "other", which cannot
+	// be parsed by JIMM - it needs a valid cloud/region to determine
+	// which controller to add the cloud to.
+	SkipJAAS(t)
```

Comment:
Do we need to install and add the microk8s config to the JAAS integration tests environment if there are no tests that are using it?